src/standalone_custom/heat_demand_profile_demandlib.py

```diff
@@ -193,13 +195,28 @@ class of the country specific workalender
 
 if __name__ == "__main__":
     plots = True
-
-    # load dummy weather
-    weather = pd.read_csv(f"era5_weather_UVTgV_{year}.csv", parse_dates=True).set_index(
-        "time"
+    weather_data_name = (
+        "uvtgv"  # "uvtgv" is processed monitored data, "era5" is ERA5 data
```

Comment:
Probably nicer to have this string defined in the head of the file, so that one does not have to search later. But you can also leave it.

Reply:
Moved it to parameters